web/locales/en.json

```diff
@@ -8,7 +8,7 @@
     },
     "onboarding-modal": {
         "view1": {
-            "header": "The Electricity Map",
+            "header": "electricityMap",
```

Comment:
should be changed in all languages. Probably a good idea to delete that line (in all languages) and hardcode the name so we don't have people translating the product name